frontend/packages/operator-lifecycle-manager/integration-tests/scenarios/global-installmode.scenario.ts

```diff
@@ -143,8 +144,8 @@ describe('Interacting with an `AllNamespaces` install mode Operator (Jaeger)', (
   it('displays metadata about Operator in the "Overview" section', async () => {
     await browser.get(`${appHost}/k8s/ns/${testName}/clusterserviceversions`);
     await crudView.isLoaded();
-    await operatorView.rowForOperator('Jaeger Tracing').click();
-    await browser.wait(until.presenceOf($('.loading-box__loaded')), 5000);
+    await operatorView.operatorNameLink('Jaeger Tracing').click();
+    await browser.wait(until.presenceOf($('.loading-box__loaded')), BROWSER_TIMEOUT);
```

Comment:
`BROWSER_TIMEOUT` is already the default right? You can just leave that off.

```suggestion
    await browser.wait(until.presenceOf($('.loading-box__loaded')));
```

Reply:
Yes, dropping, will be in next push.